components/dash-core-components/src/components/RadioItems.react.js

```diff
@@ -63,39 +74,48 @@ export default class RadioItems extends Component {
 }
 
 RadioItems.propTypes = {
+    /**
+     * An array of options, or inline dictionary of options
+     */
+    options: PropTypes.oneOfType([
+        /**
+         * Simpler `options` representation in dictionary format
+         * {`label1`: `value1`, `label2`: `value2`, ... }
+         * which equals to
+         * [{label: `label1`, value: `value1`}, {label: `label2`, value: `value2`}, ...]
+         */
+        PropTypes.object,
```

Comment:
Maybe we can use `objectOf` here to get some more type checking. String, number, bool